packages/api/cms-api/src/blob-storage/backends/file/blob-storage-file.storage.ts

```diff
@@ -76,10 +76,18 @@ export class BlobStorageFileStorage implements BlobStorageBackendInterface {
     }
 
     async getFile(folderName: string, fileName: string): Promise<NodeJS.ReadableStream> {
+        if (!(await this.fileExists(folderName, fileName))) {
+            throw new Error(`File does not exist: ${path}`);
```

Comment:
I'm not sure if throwing an error here and relying on the Admin's file previews to catch the error is the best way. We could also return `undefined` or `null` here instead. What do you think?